src/ImageSharp/Color/Color.NamedColors.cs

```diff
@@ -0,0 +1,721 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+namespace SixLabors.ImageSharp
+{
+    /// <content>
+    /// Contains static named color values.
+    /// </content>
+    public readonly partial struct Color
+    {
+        /// <summary>
+        /// Represents a <see paramref="Color"/> matching the W3C definition that has an hex value of #F0F8FF.
+        /// </summary>
+        public static readonly Color AliceBlue = FromRgba(240, 248, 255, 255);
```

Comment:
`public static readonly Color AliceBlue = Rgba32.AliceBlue;` etc would be better here. That way we're only declaring component values in one place.

Reply:
`Rgba32.AliceBlue` is created using `NamedColors<T>`. I'm not sure about the fate of the latter, it might become obsolete with #907.

Reply:
Ah you’re exactly right! I’m thinking the wrong way entirely. There’ll be no need for them at all so I think they can be removed (I don't want to obsolete classes at the moment as people ignore warnings). 

I’m gonna cancel my current review and approve it.